.ci_support/migrations/libgdal36.yaml

```diff
@@ -0,0 +1,9 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+gdal:
+- '3.6'
+libgdal:
+- '3.6'
+migrator_ts: 1668608830.3535678
```

Comment:
@SrNetoChan I wonder if this file is causing gdal 3.6 to be selected as it suggests that the gdal 3.6 migration is complete?